src/browser/sync/authentication_manager.ts

```diff
@@ -237,7 +231,6 @@ export class AuthenticationManager {
     }
 
     if (token.value && this.syncState.isNewAuth(token.value)) {
-      this.authenticate(token.value);
```

Comment:
This didn't actually do anything, will fail to send whether connection is paused or stopped.